app/src/main/java/org/mozilla/fenix/historymetadata/view/RecentlyVisited.kt

```diff
@@ -118,8 +117,7 @@ private fun RecentVisitItem(
                 onClick = { onRecentVisitClick(recentVisit) },
                 onLongClick = { menuExpanded = true }
             )
-            .size(268.dp, 56.dp)
-            .background(color = FirefoxTheme.colors.surface),
```

Comment:
This removes a reductant background modifier since it's already defined in the parent Card.